src/bundle/ui-dev/src/modules/sub-items/sub.items.module.js

```diff
@@ -184,7 +185,7 @@ export default class SubItemsModule extends Component {
 
         const shouldLoadPage = !activePageItems;
 
-        if (shouldLoadPage) {
+        if (shouldLoadPage && !this.requestInProgress) {
```

Comment:
I find the solution with a `this.requestInProgress` flag risky because e.g. what would happen if we have to perform a new search request because parameters have changed (e.g. user changes the page) but the old request is still running? I think we should rely here on the _state_ to avoid problems.